### PR TITLE
[schema]: Add SWITCH_CAPABILITY_TABLE

### PR DESCRIPTION
**Pull request by stcheng**
Signed-off-by: Shu0T1an ChenG <shuche@microsoft.com>

### CODE REVIEW

File: common/schema.h [hunk 1]
```diff
@@ -178,6 +178,7 @@ namespace swss {
 
 /***** STATE DATABASE *****/
 
+#define STATE_SWITCH_CAPABILITY                     "SWITCH_CAPABILITY_TABLE"
```
Comment (prsunny) [inline common/schema.h]: STATE_SWITCH_CAPABILITY_NAME ?

Comment (stcheng) [reply common/schema.h]: thanks. fixed

**Comment by stcheng** [comment]
retest this please

**Comment by stcheng** [comment]
retest this please

**Comment by stcheng** [comment]
retest this please

**Pull request merged**